src/common/transformations/src/ov_ops/swiglu.cpp

```diff
@@ -54,16 +57,18 @@ std::shared_ptr<Node> SwiGLU::clone_with_new_inputs(const ov::OutputVector& new_
 
 std::vector<ov::PartialShape> shape_infer(const SwiGLU* op, std::vector<ov::PartialShape> input_shapes) {
     ov::op::v1::VariadicSplit variadic_split;
-    std::vector<int64_t> axis = { op->get_axis() };
-    std::vector<int64_t> split_lengths = { op->get_split_lengths(), -1 };
+    std::vector<int64_t> axis = {op->get_axis()};
+    std::vector<int64_t> split_lengths = {op->get_split_lengths(), -1};
 
     std::unordered_map<size_t, ov::Tensor> const_data;
     const_data.emplace(1, ov::Tensor(ov::element::i64, ov::Shape{}, static_cast<void*>(axis.data())));
-    const_data.emplace(2, ov::Tensor(ov::element::i64, ov::Shape{split_lengths.size()}, static_cast<void*>(split_lengths.data())));
+    const_data.emplace(
+        2,
+        ov::Tensor(ov::element::i64, ov::Shape{split_lengths.size()}, static_cast<void*>(split_lengths.data())));
```

Comment:
```suggestion
        ov::Tensor(ov::element::i64, ov::Shape{split_lengths.size()}, split_lengths.data()));
```
Optional detail: cast should not be required.

Reply:
For now leaving as is, created a ticket 157615 for the shape_infer refactor as more changes could be applied, like moving this function to the shape_inference directory.